### ISSUE TITLE
Feature request: Clear system keyboard when clearing all clippings

### ISSUE THREAD

**Issue by CHigginbotham** [snark/jumpcut#53]
So currently you click "Clear All" and it clears all clippings but the last thing most recently in the clipboard. On the system side though, the most recently copied item remains in the clipboard until over-written by something else. It would be nice if this were cleared as well so the clipboard was completely fresh. 

> In testing I discovered that macOS ignores it if just a space is copied into the clipboard. You cannot paste anything. The idea would be to copy a space into the clipboard and then clear. This is one possibility. But you would need to have Jumpcut ignore the space in its history (not if you copy and then clear it though). However, I'm guessing there is a better way to clear the keyboard.

- Thanks! Love the app btw!

**Comment by snark** [comment]
Hm, this is slightly tricky given the way Jumpcut handles detecting new clips. You're right that maintaining the most recent item is slightly surprising; let me think about how might be best to handle this.

**Comment by CHigginbotham** [comment]
Wow awesome! it actually works. I did discover one two bugs with relating to it however.

First is #65 

Secondly: 
when selecting a different clipping than the most recently clipped item and then clearing all it does not clear the system keyboard.

Steps to reproduce:
1. Copy text string
2. Copy different text string
3. Select the string you first copied in Jumpcut (pasting it into the clipboard)
4. "Clear All"
5. You will find the system keyboard remains intact

Basically it seems to only clear the system keyboard if the system keyboard matches the most recent Jumpcut clipping.
